__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const data = require('../db/data/test-data/index');
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/seeds/seed');
+
+beforeEach(() => {
+    return db(data);
+  });
+
+  afterAll(() => {
+    if (db.end) db.end();
```

Comment:
I think you could get away with db.end() by itself here, no need for a conditional.